src/content/blog/astro-tutorial.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Announcing the Astro Tutorial
+description: 'Wrapping up Astro Docs week with our biggest Docs announcement to date: our first Astro Tutorial.'
+publishDate: 'November 4, 2022'
+socialImage: '/assets/blog/astro-tutorial/cover.png'
+coverImage: '/assets/blog/astro-tutorial/cover.png'
+lang: 'en'
+authors:
+    - sarah
+    - chris
+    - dan
+    
+---
+
+import CTA from '/src/components/CallToAction.astro'
+import BlogContentImage from '/src/components/BlogContentImage.astro'
+import community from '/src/assets/blog/docs-hacktoberfest/contributors.jpg'
+
+We hope you enjoyed Astro Docs Week, where our Docs core maintainers took control of the keyboard and our Twitter account! Each day, we posted a peek behind the scenes of life on Team Docs:
+
+- our [low-code Hacktoberfest drive](https://twitter.com/astrodotbuild/status/1587160080717434881)
+- our [custom-built translation status tracker](https://twitter.com/astrodotbuild/status/1587547028258582528)
+- our ways of publicly recognizing contributors including [Astro Badges](https://twitter.com/astrodotbuild/status/1587901468585205760) and personalized [first-time contributor celebrations](https://twitter.com/astrodotbuild/status/1588285834054148096)
```

Comment:
I know it spoils the “our, our, our” rhythm, but this sentence keeps bugging me as clunky. Are we willing to sacrifice an “our” for something like this?

```suggestion
- how we publicly recognize contributors with [Astro Badges](https://twitter.com/astrodotbuild/status/1587901468585205760) and personalized [first-time contributor celebrations](https://twitter.com/astrodotbuild/status/1588285834054148096)
```

Reply:
I’d even go as far as removing “publicly” potentially.

```suggestion
- how we recognize contributors with [Astro Badges](https://twitter.com/astrodotbuild/status/1587901468585205760) and personalized [first-time contributor celebrations](https://twitter.com/astrodotbuild/status/1588285834054148096)
```